io.github.JaredTweed.AudioToTextTranscriber.yml

```diff
@@ -0,0 +1,49 @@
+app-id: io.github.JaredTweed.AudioToTextTranscriber
+runtime: org.gnome.Platform
+runtime-version: "48"
+sdk: org.gnome.Sdk
+
+command: audio-to-text-transcriber
+
+finish-args:
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --filesystem=home
+
+modules:
+  - name: app
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/JaredTweed/Audio-To-Text-Transcriber/releases/download/v1.0.0/v1.0.0.zip
+        sha256: 5f5330de03cd20643b00c4aeb85d1e3604e7ddf674b10afe685c2d2c61fa3780
+        strip-components: 0 
+    build-commands:
+      - install -d /app/share/audio-to-text-transcriber
+      - cp -a . /app/share/audio-to-text-transcriber/
+      - ls /app/share/audio-to-text-transcriber/
+      - ls /app/share/audio-to-text-transcriber/whisper.cpp/
+      # --- remove obsolete link flag from ALL CMakeLists.txt -------------
+      - |
+          cd /app/share/audio-to-text-transcriber/whisper.cpp
+
+          # ── remove obsolete link flag everywhere ─────────────────────────
+          find . -name CMakeLists.txt -exec sed -i '/stdc++fs/d' {} \;
```

Comment:
NEVER use find + sed. it has no failure mode. Use a patch. Or better, stop requiring `stdc++fs` It has been part of stdlib for a while now.